### ISSUE TITLE
Datepicker doesn't support aria-describedby attribute

### ISSUE THREAD

**Issue by Will-Smith-007** [primefaces/primeng#18264]
### Describe the bug

```html
<p-datepicker aria-describedby="datepicker-information" />
<small id="datepicker-information">Additional information</small>
```
`aria-describedby` doesn't get applied to the child <input>.
`attr.aria-describedby` also doesn't fix the issue.

### Pull Request Link

https://github.com/primefaces/primeng/pull/18265

### Reproducer

https://stackblitz.com/edit/nc7m654p?file=src%2Fapp%2Fdatepicker-basic-demo.html

### Environment

Newly created project

### Angular version

19.0.1

### PrimeNG version

v19

### Steps to reproduce the behavior

1. Create new Angular project with PrimeNG
2. Add the attribute `aria-describedby` or `[attr.aria-describedby]` to a `p-datepicker` component.
3. Look at the `input` element of the datepicker and see that there is no `aria-decribedby` association.

### Expected behavior

The datepicker `input` should be associated with `aria-describedby`, so that a screen reader can read the description when `p-datepicker` is focused.

**Comment by Will-Smith-007** [comment]
This issue is still relevant and will be fixed by #18265. The PR was updated.
